src/main/flight/imu.c

```diff
@@ -86,10 +87,7 @@ static bool imuUpdated = false;
 #define ATTITUDE_RESET_QUIET_TIME 250000   // 250ms - gyro quiet period after disarm before attitude reset
 #define ATTITUDE_RESET_GYRO_LIMIT 15       // 15 deg/sec - gyro limit for quiet period
 #define ATTITUDE_RESET_ACTIVE_TIME 500000  // 500ms - Time to wait for attitude to converge at high gain
-#define GPS_COG_MIN_GROUNDSPEED 100        // 1.0m/s - the minimum groundspeed for a gps based IMU heading to be considered valid
-                                           // Better to have some update than none for GPS Rescue at slow return speeds
-#define GPS_COG_MAX_GROUNDSPEED 500        // 5.0m/s - Value for 'normal' 1.0 yaw IMU CogGain
-
+#define GPS_COG_MIN_GROUNDSPEED 100        // 1.0m/s - min groundspeed for GPS Heading reinitialisation etc
```

Comment:
(did you test with higher threshold value? At 1m/s, the gain is already 1.0, but CoG is probably unusable)

Reply:
Yes, I did test this carefully.  
GPS-reported heading from groundspeed is usable even at 0.5m/s.
By 1.0m/s it's definitely usable.